ftml/src/render/html/element/text.rs

```diff
@@ -72,12 +72,38 @@ pub fn render_code(
     // TODO: syntax highlighting based on 'language'
 
     ctx.html() //
-        .pre()
+        .div()
         .attr(attr!(
             "is" => "wj-code",
             "class" => &class,
         ))
         .contents(|ctx| {
-            ctx.html().code().inner(log, &contents);
+            // Panel for holding additional features
+            ctx.html()
+                .div()
+                .attr(attr!(
+                    "class" => "wj-code-panel"
+                ))
+                .contents(|ctx| {
+                    // Copy to clipboard button
+                    ctx.html().button().attr(attr!(
+                        "is" => "wj-code-copy",
+                        "type" => "button",
+                        "class" => "wj-code-copy"
+                        // would need to be localized
+                        // "title" => "Copy to Clipboard"
```

Comment:
Do `ctx.handle().get_message(log, ctx.language(), "copy-button")` and then add a stubbed case for `copy-button` in `src/render/handle.rs`.

Reply:
Done